renaming to crane operator and updating README with latest deps (#33)
konveyor/crane-operator: README.md

- mtRHO is compatible with OpenShift 4.10.7/4.10.7+ versions.
- Go 1.18 version needed to build operator

## Dependencies

> 1. mtRHO installs pipeline operator as dependency in `<installation>` namespace. (**Note:** If this operator is already installed in some other namespace than it won't be again installed by installing mtRHO. Uninstalling mtRHO will not uninstall pipeline operator as well.)

Reviewer: ```suggestion
1. mtRHO installs pipeline operator as dependency in `<installation>` namespace. (**Note:** This will not impact any existing installation of the pipeline operator. Uninstalling mtRHO later also will not impact the pipeline operator.)
```
